app/controllers/users/phone_setup_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Users
+  class PhoneSetupController < ApplicationController
+    include UserAuthenticator
+    include PhoneConfirmation
+
+    before_action :authorize_otp_setup
+    before_action :authenticate_user
+
+    def index
+      @user_phone_form = UserPhoneForm.new(current_user)
+      analytics.track_event(Analytics::USER_REGISTRATION_PHONE_SETUP_VISIT)
+    end
+
+    def create
+      @user_phone_form = UserPhoneForm.new(current_user)
+      result = @user_phone_form.submit(params[:user_phone_form])
```

Comment:
Any chance we could pull in strong parameters here to make sure we are only permitting the params we want?